README.md

```diff
@@ -194,7 +194,12 @@ name = "zoning"                              # used in the URL to reference this
   max_zoom = 18                            # maximum zoom level to include this layer
 ```
 
-\* more on PostgreSQL SSL mode [here](https://www.postgresql.org/docs/9.2/static/libpq-ssl.html). The `postgis` config also supports "ssl_cert" and "ssl_key" options are required, corresponding semantically with "PGSSLKEY" and "PGSSLCERT". These options do not check for environment variables automatically. See the section [below](#environment-variables) on injecting environment variables into the config.
+ 
+More information on PostgreSQL SSL modes can be found [here](https://www.postgresql.org/docs/9.2/static/libpq-ssl.html). 
```

Comment:
I say bump this URL to use `current`: https://www.postgresql.org/docs/current/libpq-ssl.html